tests/unit/src/test/resources/mtags-scala3/example/EndMarker.scala

```diff
@@ -0,0 +1,6 @@
+package example
+
+object EndMarker/*example.EndMarker.*/:
+  def foo/*example.EndMarker.foo().*/ =
+    1
+  end foo
```

Comment:
It feels like we should detect `example.EndMarker.foo()` here, since semanticdb does that.

Reply:
They are mostly needed for definition so that should be fine.